tests/lcs/core/interface/snapshot.livecodescript

```diff
@@ -0,0 +1,75 @@
+script "CoreInterfaceSnapshot"
+/*
+Copyright (C) 2015 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on _TestImportScreenSnapshotFromRectHandler
+    import snapshot from rectangle 100,100,500,400
+    
+    TestAssert "long id of new image is long id of it",  the long id of it is the long id of the last image
+    TestAssertBroken "long id of image is it", it is the long id of the last image
+end _TestImportScreenSnapshotFromRectHandler
+
+on TestImportScreenSnapshotFromRectNoUI
+    if the environment is not "command line" then 
+        return "SKIP TEST"
+    end if  
+    TestAssertThrow "should now throw an error", "_TestImportScreenSnapshotFromRectHandler",\
```

Comment:
actually there must be something else going on because the log file is telling me the error is thrown